exp/views/responses.py

```diff
@@ -286,6 +300,165 @@ def get_frame_data(resp: Union[Response, Dict]) -> List[FrameDataRow]:
     return frame_data_tuples
 
 
+# Helper function for building study response objects for response overview download
+def get_study_responses_csv(self, paginator, study):
+    headers = set()
+    session_list = []
+
+    for page_num in paginator.page_range:
+        page_of_responses = paginator.page(page_num)
+        for resp in page_of_responses:
+            row_data = flatten_dict(
+                {col.id: col.extractor(resp) for col in RESPONSE_COLUMNS}
+            )
+            # Add any new headers from this session
+            headers = headers | row_data.keys()
+            session_list.append(row_data)
+    header_options = set(self.request.GET.getlist("data_options"))
+    header_list = get_response_headers(header_options, headers)
+
+    return session_list, header_options, header_list
+
+
+# Helper function for taking session_data dict list and converting it to csv formatted string
+def build_overview_str(session_list, header_list):
+    for _ in session_list:
+        print(_)
+    session_strings = [
+        ",".join(
+            [
+                (str(session_row[key]) if key in session_row else "")
+                for key in header_list
+            ]
+        )
+        for session_row in session_list
+    ]
+    header_str = ",".join(header_list)
+    return "\n".join([header_str] + session_strings)
+
+
+# Helper function for aggregating child data over responses
+def get_child_overview_csv(paginator, study):
+    child_list = []
+    session_list = []
+
+    for page_num in paginator.page_range:
+        page_of_responses = paginator.page(page_num)
+        for resp in page_of_responses:
+            row_data = flatten_dict(
+                {
+                    col.id: col.extractor(resp)
+                    for col in RESPONSE_COLUMNS
+                    if col.id in CHILD_CSV_HEADERS
+                }
+            )
+            if row_data["child__global_id"] not in child_list:
+                child_list.append(row_data["child__global_id"])
+                session_list.append(row_data)
+    return session_list
+
+
+# Helper function to grab all relevant data for framedata responses for psychds download
+def get_framedata_for_psychds(paginator, study):
+    study_short_uuid = study.uuid.hex[:4]
+    response_data = []
+    variables_measured = []
+    for page_num in paginator.page_range:
+        page_of_responses = paginator.page(page_num)
+        for resp in page_of_responses:
+            response_short_uuid = resp.uuid.hex[:4]
```

Comment:
Let's use 8 characters rather than 4!  @bleonar5 how tricky is the checkbox-for-full-length-names feature?  If not too bad, I'd vote for throwing an error that instructs the user to uncheck it. 

Calculator on crash likelihoods here: https://devina.io/collision-calculator